hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
@david-yz-liu could make a different branch for you to push to so you can commit code like this that's not exactly complete for master, depending on what you two have arranged. Like the react-* ones that you see - code committed there is not complete but we push it there so everyone can look at it and can keep up to date.